docs/linuxInstall.md

```diff
@@ -1,44 +1,16 @@
 # Linux install instructions
 
-Signed packages are on their way, but in the meantime you can use the following.
-
-**NOTE**: _If you experience a problem with dependencies while installing, you may
-want to try installing `git` using the package manager for your distro._
-
-## Debian AMD64:
-To install brave using apt:
-``` 
-curl https://s3-us-west-2.amazonaws.com/brave-apt/keys.asc | sudo apt-key add -
-deb https://s3-us-west-2.amazonaws.com/brave-apt jesse main  
-```
-
-Upgrades can be done via:
-```
-apt-get update && apt-get upgrade -y
-```
-
-Alternatively you can install the deb directly but then you wont get automatic upgrades with apt
-```
-wget -O brave.deb https://laptop-updates.brave.com/latest/dev/debian64
-sudo apt-get install -y gdebi && sudo gdebi brave.deb
-```
-
-## Ubuntu AMD64:
+## Debian or Ubuntu AMD64:
 To install brave using apt:
 ``` 
 curl https://s3-us-west-2.amazonaws.com/brave-apt/keys.asc | sudo apt-key add -
```

Comment:
not part of this PR but i had to do `curl https://s3-us-west-2.amazonaws.com/brave-apt/keys.asc 2> /dev/null | sudo apt-key add -` to prevent curl from spamming the output